QuadradiusR/scripts/torus.gd

```diff
@@ -139,7 +139,12 @@ func make_move(source_tile: Tile, target_tile: Tile, is_colliding: bool = false)
 		anim.play("DestroyOpponent")
 	# TODO move the sound playing to here maybe to handle dropping the piece
 	pass
-	
+
 # TODO maybe add method destroy() which would handle aftermath
 # OR destroy_piece(torus) which would make one piece destroy the other, so it could handle
 # all the consequences
+
+
+func add_power(power_id: String, power_definition_id):
```

Comment:
`power_definition_id: String` but it's a minor issue

Reply:
This is actually intentional, as power definition id can be null (for example if it was an opponent who received it) and adding a typehint causes an error when null is passed